packages/dev/test/StakeManager.test.ts

```diff
@@ -7,12 +7,21 @@ import { StakeManagerInstance } from '../../../types/truffle-contracts'
 
 const StakeManager = artifacts.require('StakeManager')
 
-contract('StakeManager', function ([_, relayManager, anyRelayHub, owner, nonOwner]) {
+contract('StakeManager', function ([_, relayManager, anyRelayHub, owner, nonOwner, secondRelayManager]) {
   const initialUnstakeDelay = new BN(4)
   const initialStake = ether('1')
   let stakeManager: StakeManagerInstance
 
-  function testCanStake (relayManager: string): void {
+  function testNotOwnerCannotStake (): void {
+    it('should not allow not owner to stake for owned relayManager address', async function () {
```

Comment:
nitpic: the title should be "non-owner", and the member "nonOwner" (these are nouns, and in the "not owner" error its verb...
(but you really don't have to...)